src/components/PaymentInfoLine.vue

```diff
@@ -119,8 +112,12 @@ export default class PaymentInfoLine extends Vue {
         text-decoration: none;
     }
 
-    .shop-logo,
-    .identicon {
+    .account {
+        padding: 0;
+        width: auto !important;
```

Comment:
Why is `!important` required here?

Reply:
Because in `Account.vue` the width is set in a `.account.row` rule which has priority over the `.account` rule.